full/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Dazzle do not supports ARG TRIGGER_REBUILD=2
 FROM gitpod/workspace-base:latest
+FROM composer:2 AS composer
```

Comment:
**Thought**: Shouldn't these two lines be swapped? 🤔 (We want all the instructions below to happen in `gitpod/workspace-base:latest`, not in `composer:2`, except for the `COPY` but that already references `--from=composer` explicitly).